project/EnsimeBuild.scala

```diff
@@ -286,6 +287,10 @@ object EnsimeTestingBuild {
     libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.1" % Test intransitive ()
   )
 
+  lazy val testingSimple2 = testingProject("testing/simple2") settings (
```

Comment:
I was initially did it so that I could have members of `org.example2` in two modules `testing_simple` and `testing_simple2`. I could have used existing but I was hesitant to change the existing testing projects. 
But I am not using it now so I'll revert it. :)